release/serve_tests/workloads/serve_test_cluster_utils.py

```diff
@@ -57,12 +57,12 @@ def setup_anyscale_cluster():
         # to reduce spam.
         runtime_env={"env_vars": {"SERVE_ENABLE_SCALING_LOG": "0"}},
     )
-    serve_client = serve.start(http_options={"location": DeploymentMode.EveryNode})
+    serve.start(http_options={"location": DeploymentMode.EveryNode})
```

Comment:
nit: `location` is deprecated. By default, it is every node.

Reply:
good call, let me update that as well so it will keep working after 👍